### PR TITLE
feature - Hello-world component

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**</br>
Created a hello-world component
- Added simple structure of small component
- Readme information
- Added a new category for the storybook called START

**Solving issue**</br>
Fixes: #54

### CODE REVIEW

File: src/components/hello-world/hello-world.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Component, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'c-hello-world',
+  styleUrl: 'hello-world.scss',
+  shadow: true
+})
+export class helloworld {
```
Comment (helloimela) [inline src/components/hello-world/hello-world.tsx]: Class name should be CamelCase with first letter in capital

**Review by helloimela — DISMISSED** [review]

File: src/components/hello-world/hello-world.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  @Prop() PropHelloWorld: String = "hello-World";
```
Comment (helloimela) [inline src/components/hello-world/hello-world.tsx]: I prefer the property example is something that can be seen directly in the demo. This property will be translated in the element as `<c-hello-world prop-hello-world="A text" />`
Seems a bit cluttered for a simple example. What if change it to a more understandable property such as 'text', or 'name'.
So we can render something like: `<c-hello-world name="Jane Doe" /> `in the demo.
And according to stencil code guidelines, props are usually start with non-capital letters ;) So we don't mix up with classes (which started with capital letters)

**Comment by helloimela** [comment]
Because this component is a 'start' component that will be used by developers with different understanding of stencil, important to keep it structured & written in our common standards, which now follows stencil code guidelines for writing components.
Reference from stenciljs https://stenciljs.com/docs/style-guide

**Review by awinny — APPROVED** [review]
Looks good, lets add it 👍

**Pull request merged**